cirq-web/cirq_web/bloch_sphere/bloch_sphere_test.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import math
+import json
+
+import pytest
+import numpy as np
+
+import cirq_web
+
+from cirq.qis import to_valid_state_vector
+from cirq.qis.states import bloch_vector_from_state_vector
+
+
+def get_bundle_file_path():
```

Comment:
We typically (although I'm not sure it's consistent across the codebase) make methods private (`_get_bundle_file_path`) in tests unless we want them to be reused across different test files - but then we typically add them to `cirq.testing` as a test utility and write tests for them too.